### PR TITLE
175 clear all layers prevent drawing

### LINKED ISSUE #175: clear all layers prevent drawing
double check the logic of clear all layers
I cannot draw after clear all
might accidentally delete the draw interaction

### PR DESCRIPTION
**Pull request by Rashmil-1999**
This PR fixes the clear layer bug

### CODE REVIEW

File: src/components/childComponents/outputWidgets/PopDislocation.jsx [hunk 1]
```diff
@@ -41,6 +41,20 @@ export default function PopDislocation(props) {
 		datawolfDatasetConfigItems.map(async (datawolfDatasetConfigItem) => {
 			const vName = datawolfDatasetConfigItem.datasets.variableName;
 			if (vName === "bldgMaxDamageStateINCOREId") {
+				const bldgMaxDamageStateINCOREId = await getOutputFileContent(
```
Comment (longshuicy) [inline src/components/childComponents/outputWidgets/PopDislocation.jsx]: i think those accidently got committed in...

File: src/components/childComponents/outputWidgets/HHS.jsx [hunk 2]
```diff
@@ -99,7 +93,6 @@ export default function HHS(props) {
 								{HHSBarChartContent ? (
 									<HHSBarChart
 										data={HHSBarChartContent[t]}
-										height={"25em"}
 										xAxisLegend="Housing Recovery Stage"
 										yAxisLegend="Frequency"
 									/>
```
Comment (longshuicy) [inline src/components/childComponents/outputWidgets/HHS.jsx]: same here can you revert the change of these? There shouldn't be any diff here

**Review by longshuicy — CHANGES_REQUESTED** [review]
Could you bump version up to 0.5.1 (I published 0.5.0 last Thursday)
And add an entry in the changelog please. Thanks!

File: src/components/childComponents/outputWidgets/HHS.jsx (hunk 2 above)
Comment (Rashmil-1999) [reply src/components/childComponents/outputWidgets/HHS.jsx]: I actually pulled develop and merged it.

Comment (Rashmil-1999) [reply src/components/childComponents/outputWidgets/HHS.jsx]: There were conflicts with this branch and develop

Comment (longshuicy) [reply src/components/childComponents/outputWidgets/HHS.jsx]: hmm then when you pointing back to develop there shouldn't be those changes. You might have accepted the changes on your branch instead of the develop branch when resolving the conflicts?

Comment (longshuicy) [reply src/components/childComponents/outputWidgets/HHS.jsx]: The red highlighted parts are actually needed :-(

Comment (Rashmil-1999) [reply src/components/childComponents/outputWidgets/HHS.jsx]: I copied the develop file and update it. There was some issue when I was resolving conflicts...

Comment (Rashmil-1999) [reply src/components/childComponents/outputWidgets/HHS.jsx]: now there will only be formatting changes.

**Review by longshuicy — APPROVED** [review]
